### PR TITLE
Create Task3

### PR DESCRIPTION
**Pull request by Artur4ik11**
For your convenience - added 'print' at the beginning with the name of the program and its functions

### CODE REVIEW

File: Tasks/Mardoyan/Task3/Task3_1.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+print('Hello! I can help you count the vowels in your string.')
+x = input('Input you string(on English)\n')
+vowels = 0
+for i in x:
+    letter = i.lower()
+    if letter == "a" or letter == "i" or\
+       letter == "u" or letter == "e" or\
+       letter == "y" or letter == "o":
+       vowels += 1
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Mardoyan/Task3/Task3_1.py:9]: would be better to have all letter stored in a list and perform check with the "in" operator

File: Tasks/Mardoyan/Task3/Task3_6.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+print('Hello!I can return the string with all vowels removed.')
+x = input('Input you string(on English)\n')
+b =''
+for i in x:
+    letter = i.lower()
+    if letter != "a" and letter != "i" and\
+       letter != "u" and letter != "e" and\
+       letter != "y" and letter != "o":
+       b += letter
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Mardoyan/Task3/Task3_6.py:9]: same remark about list goes here

File: Tasks/Mardoyan/Task3/Task3_10.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+print('Hello!I can return the largest prime number in your list.')
+x = input('Input you list of numbers(Throught "space")\n')
+prime_list = []
+max = 0
+x = x.split(' ')
+for i in x:
+    if (int(i) % 2 != 0 and int(i) % 3 != 0) or ( int(i) == 2 or int(i) == 3):
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Mardoyan/Task3/Task3_10.py:7]: only 2 and 3 and no other numbers? not good enough

**Review by MikitaTsiarentsyeu — APPROVED** [review]
good work, the score is 90, because 10th task is incomplete

File: Tasks/Mardoyan/Task3/Task3_10.py (hunk 3 above)
Comment (Artur4ik11) [reply Tasks/Mardoyan/Task3/Task3_10.py:7]: I don't understand enough why it is incorrectly.The program works great.
I could solve it through a nested loop, but in this version it works

**Pull request merged**